q2_quality_control/decontam.py

```diff
@@ -14,36 +14,49 @@
 from ._utilities import _run_command
 
 
-def _check_column_inputs_helper(table, metadata,
-                                prev_control_column,
-                                prev_control_indicator):
-    if prev_control_column not in metadata.columns:
-        raise ValueError('Prevalence column not found, please '
-                         'select from:\n'
-                         + str(', '.join(metadata.columns)))
+def _check_prev_inputs(table, metadata, prev_control_column,
+                       prev_control_indicator, method):
+    if prev_control_column is None or prev_control_indicator is None:
+        raise ValueError('For the ' + str(method) + 'method '
+                         'please check that input parameters'
+                         ' --prev-control-column and '
+                         '--p-prev-control-indicator are being utilized')
     else:
```

Comment:
These `else` statements aren't necessary because of the `ValueError`s being raised. Removing these will also help with nesting.

Reply:
addressed in commit [f31210b](https://github.com/qiime2/q2-quality-control/pull/88/commits/f31210b1b4524e53c8545435910ecb106d40fc8d)